specification/securityinsights/data-plane/Microsoft.SecurityInsights/preview/2022-12-01-preview/ThreatIntelligence.json

```diff
@@ -0,0 +1,187 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Sentinel Upload Indicators API",
+    "description": "Spec for Microsoft Sentinel Upload Indicators API.",
+    "version": "2022-12-01-preview"
+  },
+  "host": "sentinelus.azure-api.net",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/workspaces/{workspaceId}/threat-intelligence/indicators:upload": {
+      "post": {
+        "x-ms-examples": {
+          "Upload Threat Intelligence indicators": {
+            "$ref": "./examples/threatintelligence/UploadThreatIntelligence.json"
+          }
+        },
+        "summary": "Upload the indicators to the workspace",
+        "description": "Upload the list of indicators to the workspace specified",
+        "operationId": "threat-intelligence/indicators:upload",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "workspaceId",
+            "description": "The workspace ID for the indicators to be uploaded.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Body of the request with SourceSystem and array of indicators",
```

Comment:
"Body of the request with an array of indicators and other properties"
Let's not too specific here, otherwise, we need to update this description when we add more properties